Following function touches Cairo's internal API and breaks build when using
system Cairo. Fortunately it isn't used, so it can be removed :)
--- gfx/thebes/public/gfxMatrix.h.orig	2007-03-05 22:28:55.000000000 +0200
+++ gfx/thebes/public/gfxMatrix.h	2007-03-05 22:29:05.000000000 +0200
@@ -165,8 +165,6 @@
      */
     gfxRect Transform(const gfxRect& rect) const;
 
-    gfxRect TransformBounds(const gfxRect& rect) const;
-
     /**
      * Returns the translation component of this matrix.
      */
--- gfx/thebes/src/gfxMatrix.cpp.orig	2007-03-05 22:28:16.000000000 +0200
+++ gfx/thebes/src/gfxMatrix.cpp	2007-03-05 22:28:39.000000000 +0200
@@ -105,15 +105,3 @@
     return gfxRect(Transform(rect.pos), Transform(rect.size));
 }
 
-gfxRect
-gfxMatrix::TransformBounds(const gfxRect& rect) const
-{
-    double x0 = rect.pos.x;
-    double y0 = rect.pos.y;
-    double x1 = rect.pos.x + rect.size.width;
-    double y1 = rect.pos.y + rect.size.height;
-
-    cairo_matrix_transform_bounding_box(CONST_CAIRO_MATRIX(this), &x0, &y0, &x1, &y1, NULL);
-
-    return gfxRect(x0, y0, x1 - x0, y1 - y0);
-}
